axelrod/strategies/grudger.py

```diff
@@ -62,3 +62,17 @@ def strategy(self, opponent):
         if 'C' in opponent.history:
             return 'C'
         return 'D'
+
+class Davis(Player):
+    """A player starts by cooperating for 10 rounds then plays Grudger, defecting if at any point the opponent has defected."""
+
+    name = 'Davis'
+    memoryone = False  # Long memory
+
+    def strategy(self, opponent):
+        """Begins by playing C, then plays D for the remaining rounds if the opponent ever plays D."""
+        if len(self.history) < 10:
+            return 'C'
+        if 'D' in opponent.history:
```

Comment:
I think that technically (although this could be about interpreting the paper) this should be:

```
if 'D' in opponent.history[10:]
```

So that you 'play grudger' after the first ten moves.